ngraph/core/include/openvino/op/slice.hpp

```diff
@@ -14,7 +14,7 @@ namespace v8 {
 ///
 class OPENVINO_API Slice : public Op {
 public:
-    OPENVINO_RTTI_DECLARATION;
+    OPENVINO_OP("Slice", "opset8");
```

Comment:
Don't we need specify version `8` here? Like for other new opset8 ops: `OPENVINO_OP("AdaptiveMaxPool", "opset8", op::Op, 8);`
```suggestion
    OPENVINO_OP("Slice", "opset8", op::Op, 8);
```

Reply:
I created a separate PR for operations: https://github.com/openvinotoolkit/openvino/pull/7573

I removed version from all opset8 operations. we have a plan to remove verson from `DiscreteTypeInfo` after several releases.